app/src/ui/app.tsx

```diff
@@ -583,6 +583,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
+      const actionOnDoubleClick = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
```

Comment:
Or we could define a string literal further up to match the expected set of values:

```ts
type AppleActionOnDoubleClickOption = 'Maximize' | 'Minimize' | 'None'
```

I gather this value can change while the app is running, so we should check this at click time rather than at launch.